### DISCUSSION TITLE
How to get the token balance in Solana

### DISCUSSION THREAD

**Discussion by CuteGuy99** [community/community#143605] [Programming Help] [answered]
### Body

Hey guys
I am going to get the token balance in Solana ecosystem.
I am using @solana/web3.js module
how can i get?
Thanks

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Octorcat** [comment]
Hi @CuteGuy99 
You can get like this
const conn = new Connection(rpcEndpoint);
const accountInfo = await conn.getTokenAccountBalance(tokenAccount);
console.log(accountInfo.value.amount);

**Reply by CuteGuy99** [reply]
tokenAccount is publick address?

**Reply by Octorcat** [reply, marked as answer]
sure

**Comment by jenn42022** [comment]
I won 1400 on stake and now it's on Solana and usdc how do I get my money
